docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 version: "3.4"
 services:
-  lodestar:
+  beacon_node:
     build: .
+    image: lodestar
     restart: always
     volumes:
-      - lodestar:/root/.local/share/lodestar
-    command: beacon
+      - beacon_node:/data
+    env_file: .env
     ports:
-      - "9000:9000"
+      - "9000:9000" # P2P port
+      - "9596:9596" # REST API port
+    command: beacon --rootDir /data --api.rest.host 0.0.0.0
```

Comment:
it doesn't seem `NETWORK` env variable is carried all the way to the container, should we do `--network ${NETWORK}` here?

also metrics is disabled by default, I think we need to enable it here

Reply:
Oh right! The ENV has to be prefixed by `LODESTAR_` to be picked up by yargs